content/articles/2019-05-27-Bonding-period-ends.md

```diff
@@ -0,0 +1,82 @@
+Title: Coding phase starts!
+Date: 2019-05-27 11:30
+Category: GSCOC
+Tags: GSOC, GSOC19, poliastro
+slug: 2019-05-27-poliastro-gsoc-jorge
```

Comment:
I recommend:

```suggestion
slug: 2019-05-27-coding-phase-starts
```

And also renaming the file

Reply:
Oh sory! I thought this section called `slug` was something related to the RSS :sweat: